Source/Events.Store/IMetrics.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Prometheus;
+
+namespace Dolittle.Runtime.Events.Store
+{
+    /// <summary>
+    /// Defines the metrics related to the <see cref="IEventStore"/>.
+    /// </summary>
+    public interface IMetrics
+    {
+        /// <summary>
+        /// Gets the counter used for counting number of committed events.
+        /// </summary>
+        Counter CommittedEvents { get; }
+
+        /// <summary>
+        /// Gets the counter used for counting number of committed aggregate events and committed events.
+        /// </summary>
+        Counter CommittedAggregateEvents { get; }
+
+        /// <summary>
+        /// Gets the counter used for counting number of failed committed events.
+        /// </summary>
+        Counter FailedEvents { get; }
+
+        /// <summary>
+        /// Gets the counter used for counting number of failed committed aggregate events.
+        /// </summary>
+        Counter FailedAggregateEvents { get; }
+
+        /// <summary>
+        /// Increment number of failed uncommitted events based on an event for labelling.
+        /// </summary>
+        /// <param name="events">The <see cref="UncommittedEvents"/> used for labelling.</param>
+        void IncrementFailedEvents(UncommittedEvents events);
+
+        /// <summary>
+        /// Increment number of failed uncommitted aggregate events based on an event for labelling.
+        /// </summary>
+        /// <param name="events">The <see cref="UncommittedAggregateEvents"/> with all events and context used for labelling.</param>
+        void IncrementFailedAggregateEvents(UncommittedAggregateEvents events);
+
+        /// <summary>
+        /// Increment number of committed events based on an event for labelling.
+        /// </summary>
+        /// <param name="event">The <see cref="CommittedEvent"/> used for labelling.</param>
+        void IncrementCommittedEvents(CommittedEvent @event);
+
+        /// <summary>
+        /// Increment number of committed aggregate events and committed events based on an
+        /// event for labelling.
+        /// </summary>
+        /// <param name="event">The <see cref="CommittedAggregateEvent"/> used for labelling.</param>
+        void IncrementCommittedAggregateEvents(CommittedAggregateEvent @event);
```

Comment:
The APIs on these should be similar IMO. Now the parameter for incrementing failed commits is plural while for non-failures its not. I suggest plural (Committed(Aggregate)Events, pretty sure we have a class for that?)